service/service.go

```diff
@@ -72,6 +82,10 @@ func newService(set *settings) (*service, error) {
 		return nil, fmt.Errorf("cannot build pipelines: %w", err)
 	}
 
+	if err = telemetry.RegisterProcessMetrics(srv.colTelemetry.ocRegistry, getBallastSize(srv.host)); err != nil {
```

Comment:
Add a comment to explain why this is after `Build`.

Reply:
Done.